Makefile

```diff
@@ -217,6 +220,7 @@ build-testnode : $(IELE_TEST_VM) $(IELE_TEST_CLIENT)
 KOMPILE=kompile
 
 KOMPILE_INCLUDE_OPTS := -ccopt -I -ccopt $(PLUGIN)/plugin-c -ccopt -I -ccopt $(PROTO) -ccopt -I -ccopt $(BUILD_DIR)/plugin-node -ccopt -I -ccopt $(LOCAL_INCLUDE)
+KOMPILE_INCLUDE_OPTS += -I $(IELE_DIR)
```

Comment:
Why line split?

Also, maybe consider improving the first line here with `$(addprefix -ccopt , $(LLVM_KOMPILE_INCLUDES))`, and define the `LLVM_KOMPILE_INCLUDES` variable above.

Reply:
The two lines represent two different classes of arguments, as well as keeping line length more sane looking. The addprefix option is a good idea though, should be useful even without the variable.